src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -299,18 +744,47 @@
 		"sourcePath" : "log.sagebase.org/aclSnapshots/",
 		"tableDescriptor": {
 			"name": "aclsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"change_type": "string",
-				"snapshot_timestamp": "timestamp",
-				"owner_id": "bigint",
-				"owner_type": "string",
-				"created_on": "timestamp",
-				"resource_access": "array<struct<principalId:bigint,accessType:array<string>>>"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of access-control-list. Snapshots are taken when acl is created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
+			"columns":[{
+				"name": "change_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when change(created/updated/deleted) on acl is pushed to the queue for snapshotting."
+			},
+			{
+				"name": "change_type",
+				"type" : "string",
+				"comment" : "TThe type of changed that occurred on acl, e.g., CREATE, UPDATE, DELETE."
+			},
+			{
+				"name": "snapshot_timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the snapshot was taken (It is usually after the change happened)."
 			},
+			{
+				"name": "owner_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the Synapse object to which the acl is applied."
+			},
+			{
+				"name": "owner_type",
+				"type" : "string",
+				"comment" : "The type of the Synapse object that the acl is affecting, .e.g., ENTITY, or TEAM."
```

Comment:
Are there a list of owner types? ENTITY vs FILE?

Reply:
There is a list of owner type but we can not add it in description because of restriction of 255 char length of description.